<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_receiveArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>receive</string>
   </property>
   <widget class="QRadioButton" name="radioButton_receiveArea_hexOascii">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>hex</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_receive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_sendArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>send</string>
   </property>
   <widget class="QTextEdit" name="textEdit_sendArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_sendArea_hexOascii">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>hex</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendArea_send">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>串口刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_comRefresh">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>可用串口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_comPort">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_comBaudRate">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>串口操作</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_comOpen">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_comClose">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>260</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_comInductor">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="PlotWidget" name="graphWidget" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearFigure">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>380</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>clear Figure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_size">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox_sendArea</zorder>
  <zorder>groupBox_receiveArea</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_comRefresh</zorder>
  <zorder>label_2</zorder>
  <zorder>comboBox_comPort</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox_comBaudRate</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_comOpen</zorder>
  <zorder>pushButton_comClose</zorder>
  <zorder>label_comInductor</zorder>
  <zorder>graphWidget</zorder>
  <zorder>pushButton_clearFigure</zorder>
  <zorder>label_size</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
